spec/features/user_submits_choices_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+feature "user submits answer choices for a question", %{
+  As an indecisive user
+  I want to submit potential answer choices for my question
+  So that I can receive a relevant response
+
+  Acceptance Criteria:
+  - [X] When submitting a question, I will see a form to enter answer choices
+        for that question
+  - [X] If I submit an invalid input, I will be notified of errors
+  - [X] I may submit up to 3 answer choices
+  - [X] Upon successfully entering my answer choices, I will be notified
+        that my answer choices were submitted
+  - [ ] I cannot submit the same answer choice twice for the same question
+  - [ ] I will be taken to a page where I will receive a decision for
+        my question
+} do
+  context "User submits question and" do
+    let!(:question) { FactoryGirl.create(:question) }
+    let!(:answer_fields) {[
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.